node/src/components/consensus/protocols/highway.rs

```diff
@@ -1046,3 +1090,12 @@ where
         self.highway.next_round_length()
     }
 }
+
+// This can be derived entirely from the chainspec core_config.
+pub fn min_rounds_per_era(
```

Comment:
Maybe add an additional doc comment on what this function does, just to be sure.